oolushola/Banka
# #[165186589]Integrate continous integration with TravisCI

#### What does this PR do?
build the project and run tests

#### Description of Task to be completed?
Have TravisCI integrated to the repo

#### How should this be manually tested?
After cloning the repo, cd into it and from the terminal RUN 'npm run test' to check the build status

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165186589](https://www.pivotaltracker.com/story/show/165186589)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
